### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by pgnikhil** [FABLE-3DXRD/fabian#25]
Hi! 
I was trying to install fabian along with xrd_simulator and ImageD11.
I am not sure if it is a problem with the install, but I am unable to open or run fabian.py, it gives me an error stating unable to import Tkinter. Initially, the error was a print statement, which I corrected by editing the python file.

I am guessing it is an issue with the existing python2.7 and 3.10 versions installed in Ubuntu by default, that the program maybe launching it with python3 instead of python2.  

Furthermore, I installed Tkinter with apt-get as well, but I am unable to launch it.

Is it possible to have a procedure for install on Ubuntu 22.04 with complete dependency requirements?
Thank you!

**Comment by jadball** [comment]
Hi - are you able to provide the full error message you receive when you try to run fabian? This will help us to troubleshoot further. I've just set up a standalone Python 3.11 environment and it works fine, after installing the dependencies it complains about.

**Comment by jonwright** [comment]
See also https://github.com/orgs/dmi-3D/discussions/37

**Comment by smerkel** [comment]
We documented on how to get a run installation of fabian (and other packages) on a mac here: [http://multigrain.texture.rocks/doku.php?id=software:gettingstartedmax](http://multigrain.texture.rocks/doku.php?id=software:gettingstartedmax) 

Installing anaconda and a few additional packages usually helps.

**Comment by pgnikhil** [comment]
@jonwright, @smerkel @jadball thank you all for the very quick replies.
Initially, I tried the http://multigrain.texture.rocks/doku.php?id=software:gettingstartedmax and it was still failing somehow and made me realize it could be an ubuntu or prior installation error.

I did a fresh ubuntu reinstall, and followed the same procedure and it works flawlessly!

Thanks a lot!
(This install was for another colleague)


> We documented on how to get a run installation of fabian (and other packages) on a mac here: http://multigrain.texture.rocks/doku.php?id=software:gettingstartedmax
> 
> Installing anaconda and a few additional packages usually helps.
